buildrunner/cli.py

```diff
@@ -161,6 +161,14 @@ def parse_args(argv):
              'prevent clutter on build machines)',
     )
 
+    parser.add_argument(
+        '--clean-cache',
+        default=False,
+        action='store_true',
+        dest='clean_cache',
+        help='Clean local caches (~/.buildrunner/caches)',
```

Comment:
Change `Clean local caches (~/.buildrunner/caches)` -> `Clean local caches as define by global config "caches-root" which defaults to ~/.buildrunner/caches`

It would be extra nice if `~/.buildrunner/caches` was read from a variable instead of hardcoded so if it changes in the future less (ideally one) places in code would need to be updated.